llvm/unittests/Analysis/TargetLibraryInfoTest.cpp

```diff
@@ -281,6 +281,9 @@ TEST_F(TargetLibraryInfoTest, ValidProto) {
       "declare float @remainderf(float, float)\n"
       "declare x86_fp80 @remainderl(x86_fp80, x86_fp80)\n"
       "declare i32 @remove(i8*)\n"
+      "declare double @remquo(double, double, i32*)\n"
+      "declare float @remquof(float, float, i32*)\n"
+      "declare x86_fp80 @remquol(x86_fp80, x86_fp80, i32*)\n"
```

Comment:
Still using typed pointers?

Reply:
We need an NFC patch to clean up these typed pointers :)